extensions/wikia/Wall/WallHooksHelper.class.php

```diff
@@ -1671,7 +1675,7 @@ static public function onFilePageImageUsageSingleLink( &$link, &$element ) {
 			$threadLink = Linker::link( $titleObj, htmlspecialchars( $titleData['articleTitleTxt'] ),
 				[ 'title' => $titleData['articleTitleTxt'] ] );
 
-			$link = wfMessage( static::getMessagePrefix( $namespace ) . '-thread-group' )
+			$link = wfMessage( static::getMessagePrefix( NS_USER_WALL_MESSAGE ) . '-thread-group' )
```

Comment:
Var ```$namespace``` was not defined, so whatever this code does probably wasn't working.  Added in the NS  being checked for on L1672.